lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -457,6 +498,137 @@ def _check_job_arguments(self, jobs, has_parent_job=False):
             # although most likely jobs aren't operating on the same object
             self._check_ws_objects(source_objects=job.get(_SOURCE_WS_OBJECTS))
 
+    def retry_multiple(
+        self, job_ids, as_admin=False
+    ) -> List[Dict[str, Union[str, Any]]]:
+        """
+        #TODO Add new job requirements/cgroups as an optional param
+        :param job_ids: The list of jobs to retry
+        :param as_admin: Run with admin permission
+        :return: The child job ids that have been retried or errors
+        """
+        retried_jobs = []
+        for job_id in job_ids:
+            try:
+                retried_jobs.append(self.retry(job_id, as_admin=as_admin))
+            except Exception as e:
+                retried_jobs.append({"job_id": job_id, "error": f"{e}"})
```

Comment:
Does this include the stacktrace? My preference, I think, would be for this field to include an error string and then log the job id and stacktrace.

Reply:
No it does not include the stacktrace, it is just an error string. 
Is it really useful for us to see this in the logs in the case of when theres a bad job record id, or a permission issue? I can see this flooding the logs with those.. I can see where it could be useful when theres a job submission issue or db issue.

Reply:
Added an error statement in `_db_update_failure`

Reply:
That's a good point. Is there a reasonable way to distinguish errors that are logworthy?

Reply:
Oh, you're saying logworthy errors are the ones you catch in that method. OK, makes sense.